### PR TITLE
HOTFIX language popup

### PR DESCRIPTION
**Pull request by mathiasselleslach**

### CODE REVIEW

**Comment by mathiasselleslach** [comment]
@planctus and @haringsrob ,

This is the hotfix PR for the language popup as discussed

File: nexteuropa_language_selector_site_popup/nexteuropa_language_selector_site_popup.module [hunk 1]
```diff
@@ -64,7 +69,8 @@ function nexteuropa_language_selector_site_popup_preprocess_splash(&$variables)
   // Add enabled languages, if not in blacklist. Add all these languages to our
   // processed list.
   foreach ($languages[1] as $lang) {
-    if (!in_array($lang->prefix, $languages_blacklist)) {
+    $node_tranlated = isset($nid) ? _dt_core_is_node_translated($nid, $lang->language) : FALSE;
```
Comment (planctus) [inline nexteuropa_language_selector_site_popup/nexteuropa_language_selector_site_popup.module]: @mathiasselleslach, how could we have this in the theme tools? this is referring to a function belonging to dt_core, if that is a function that needs to be shared it has to be moved in one of our tools.

File: nexteuropa_language_selector_site_popup/nexteuropa_language_selector_site_popup.module [hunk 2]
```diff
@@ -47,6 +47,11 @@ function nexteuropa_language_selector_site_popup_preprocess_splash(&$variables)
     if (strpos($href, 'node/') == 0 &&
         function_exists('translation_path_get_translations')) {
       $translations = translation_path_get_translations($href);
+      if (preg_match('/(node)\/(\d+)/', $_GET['destination'], $matches)) {
```
Comment (planctus) [inline nexteuropa_language_selector_site_popup/nexteuropa_language_selector_site_popup.module]: This condition looks pretty redundant...
you have a first condition about the preg_match returned value, this condition is enough, i would say, if it returns true that also means that $matches is populated and it has three values, the second is "node" (you query looks only for that) , the third is "numeric" (same as before), no reason for checking further, basically.

**Comment by haringsrob** [comment]
It seems clear to me that this needs more work if we directly implement this into the theme tools as this is more specific to the site functionality rather then the theme.

We will apply this as a patch within the DT site so that we have more time to think about how to approach this issue from the theme (with alters or another option) Please also provide your input/ideas.

**Pull request closed without merging**